broker/cloud_run/lsst/ps_to_storage/main.py

```diff
@@ -95,60 +95,62 @@ def run():
 def store_alert_data(envelope) -> None:
     """Uploads the msg data bytes to a GCP storage bucket."""
 
+    # create an alert object from the envelope
+    alert = _unpack_alert(envelope)
```

Comment:
This is so painful here. I opened https://github.com/mwvgroup/pittgoogle-client/pull/81. Can we try using that so that this can be:

```suggestion
    alert = pittgoogle.Alert.from_cloud_run(envelope, 'lsst')
```

Then we ought to be able to drop `_unpack_alert`, all related code, and the kafka dependency from here.